README.md

```diff
@@ -1,6 +1,106 @@
+# Logstash Filter Aggregate Documentation
```

Comment:
I was more on the idea of keeping most of the sections in the original README, like the how to contribute + develop. This documentation is usually nice for people who want to contribute :+1:  ... what do you think ?  might be in another file ?